apps/front-client/src/features/courses/hooks/useGetUserCourse.ts

```diff
@@ -0,0 +1,50 @@
+import { Chapter } from '@prisma/client';
+import { useQuery } from '@tanstack/react-query';
+import { useSession } from 'next-auth/react';
+
+const COURSE_KEY = 'course';
+
+interface CouseI {
```

Comment:
Plus propre dans un dossier types nan ? Pour éventuellement pouvoir l'utiliser dans le reste de la feature.
```suggestion
interface CourseResponse {
```

Reply:
why not